store/conf/attrs/zimbra-attrs.xml

```diff
@@ -9629,4 +9629,13 @@ TODO: delete them permanently from here
     <desc>Account thumbnail photo</desc>
 </attr>
 
+<attr id="3022" name="zimbraSmtpStartTlsMode" type="enum" value="on,off,only" cardinality="single" optionalIn="globalConfig,server,domain" flags="serverInherited" since="8.8.6">
+  <globalConfigValue>off</globalConfigValue>
```

Comment:
I think we want to default to **only** for new installs and **on** for upgrades (would require an upgrade step for this - assuming we don't want to default to **only** for upgrades as well).

Reply:
Changed globalConfigValue to "only" and add globalConfigValueUpgrade as "off".